Frontend/src/Components/SystemPower/SystemPower.js

```diff
@@ -0,0 +1,92 @@
+import {Flex, useColorMode} from "@chakra-ui/react";
+import RangeRow from "../Shared/RangeRow";
+import CONSTANTS from "../../data-constants.json";
+import getColor from "../Shared/colors";
+
+export default function SystemPower(props) {
+    const { colorMode } = useColorMode();
+    const headerBg = getColor("header", colorMode);
+
+    return (
+        <Flex flex="auto" direction="column" overflowY="scroll">
+            <RangeRow
+                dataTitle="BMS Input Voltage"
+                dataValue={props.data?.bms_input_voltage[0] ?? -1.0}
+                dataConstant={CONSTANTS.bms_input_voltage}
+                DecimalPoint='3'
+
+                bg={headerBg}
+            />
+            <RangeRow
+                dataTitle="Main VBus Curr"
+                dataValue={props.data?.main_vbus_current[0] ?? -1.0}
+                dataConstant={CONSTANTS.main_vbus_current}
+                DecimalPoint='3'
+            />
+            <RangeRow
+                dataTitle="Main VBus"
+                dataValue={props.data?.main_vbus[0] ?? -1.0}
+                dataConstant={CONSTANTS.main_vbus}
+                DecimalPoint='3'
+                
+                bg={headerBg}
+            />
+            <RangeRow
+                dataTitle="Main 5V Bus"
+                dataValue={props.data?.main_5V_bus[0] ?? -1.0}
+                dataConstant={CONSTANTS.main_5V_bus}
+                DecimalPoint='3'
+            />
+            <RangeRow
+                dataTitle="Main 12V Bus"
+                dataValue={props.data?.main_12V_bus[0] ?? -1.0}
+                dataConstant={CONSTANTS.main_12V_bus}
+                DecimalPoint='3'
+                
+                bg={headerBg}
+            />
+            <RangeRow
+                dataTitle="Driver VBus Curr"
```

Comment:
```suggestion
                dataTitle="Driver VBus Current"
```
Same comment as above: Let's expand this to say "Current."